File: Selenium/Activities/Activity3_2.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3_2 {
    public static void main(String[] args)
    {
        WebDriver driver = new FirefoxDriver();

//Open a new browser to https://v1.training-support.net/selenium/login-form
        driver.get("https://v1.training-support.net/selenium/login-form");

//Get the title of the page and print it to the console.
        System.out.println("Title of Page is " + driver.getTitle());

//Find the username field using any locator and enter "admin" into it.
        driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");

//Find the password field using any locator and enter "password" into it.
        driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");

//Find the "Log in" button using any locator and click it.
        driver.findElement(By.xpath("//button[@onclick='signIn()']")).click();

        String a = driver.findElement(By.xpath("//div[@id ='action-confirmation']")).getText();
        System.out.println("New Page message is " + a);

        driver.quit();
    }
}

File: Selenium/Activities/Activity3_2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:

    driver.get("https://v1.training-support.net/selenium/login-form")

    print(f"Title of the page is {driver.title}")

    driver.find_element(By.XPATH, "//input[@id='username']").send_keys("admin")
    driver.find_element(By.XPATH, "//input[@id='password']").send_keys("password")
    driver.find_element(By.XPATH, "//button[@onclick='signIn()']").click()
    

    print(f"Title of new page is {driver.find_element(By.XPATH, "//div[@id ='action-confirmation']").text}")

    driver.quit()

